pyvista/core/filters/data_set.py

```diff
@@ -1801,7 +1801,9 @@ def outline_corners(  # type: ignore[misc]
         return wrap(alg.GetOutputDataObject(0))
 
     def extract_geometry(  # type: ignore[misc]
-        self: ConcreteDataSetType, extent: Sequence[float] | None = None, progress_bar: bool = False
+        self: ConcreteDataSetType,
+        extent: Sequence[float] | None = None,
```

Comment:
Is the policy is to use VectorLike instead of Sequence?
```suggestion
        extent: VectorLike[float] | None = None,
```

Reply:
VectorLike is more general and is consistent with other changes, so I think this is good practice. Will likely need some additional minor changes to satisfy `mypy` though.